packages/react-utilities/src/hooks/useOnClickOutside.ts

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { useEventCallback } from './useEventCallback';
+
+export const useOnClickOutside = (options: {
```

Comment:
```suggestion
/**
 * Handles clicks outside of passed refs.
 */
export const useOnClickOutside = (options: {
```

nit: Let's add some description